src/dmn/trn_agent_xdp_usr.c

```diff
@@ -279,7 +318,8 @@ int trn_agent_bpf_maps_init(struct agent_user_metadata_t *md)
 	md->conn_track_cache = bpf_map__next(md->ing_vsip_except_map, md->obj);
 
 	if (!md->jmp_table_map || !md->agentmetadata_map ||
-	    !md->endpoints_map || !md->xdpcap_hook_map ||
+	    !md->endpoints_map || !md->packet_metadata_map || 
+		!md->xdpcap_hook_map ||
```

Comment:
nit: please fix the alignment. Use whitespaces, no tabs.
Can this check be added to the end after conn_track_cache instead of adding a line here?

Reply:
Fixed.